src/Appwrite/Utopia/Response/Model/AuthProvider.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class AuthProvider extends Model
+{
+    /**
+     * @var bool
+     */
+    protected bool $public = false;
```

Comment:
All of the new models should be `public` (show in docs and specs) as we expose them through the API 

True is the default in `Model` so we can just remove the redeclarations

Reply:
removed from all models

Reply:
haven't removed it from auth provider model since it was there earlier